docs/units/FAIR-generics/topics/1-why-fair.md

```diff
@@ -125,7 +212,7 @@ references:
   - title: The FAIR principles of data management
     url: https://genestack.com/resources/library/the-fair-principles-of-data-management/
     additionalInformation: section about stakeholders
-  - title: "Implementing FAIR in data sharing: who are the stakeholders and what are their responsibilities?"
+  - title: Implementing FAIR in data sharing: who are the stakeholders and what are their responsibilities?
```

Comment:
As above, check if the ":" causes issues here.

Reply:
Maybe @martijnkersloot knows (and can adjust if necessary)